practice-project-bryce/src/main/java/frc/robot/Constants.java

```diff
@@ -13,7 +15,110 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
+  public static class OPERATOR_CONSTANTS {
+    public static final int K_DRIVER_CONTROLLER_PORT = 0;
+  }
+
+  public static class CAN_ID {
+    public static final int TOP_INTAKE_MOTOR_ID = 23;
+    public static final int BOTTOM_INTAKE_MOTOR_ID = 24;
+
+    public static final int TOP_SHOOTER_MOTOR_ID = 25;
+    public static final int BOTTOM_SHOOTER_MOTOR_ID = 26;
+
+    public static final int PIVOT_MOTOR_ID = 29;
+  }
+
+  public static class SHOOTER {
+    public static final IdleMode IDLE_MODE = IdleMode.kCoast;
+
+    public static final boolean TOP_MOTOR_INVERTED = false;
+    public static final boolean BOTTOM_MOTOR_INVERTED = true;
+
+    public static final int TOP_MOTOR_STALL_LIMIT_AMPS = 50;
+    public static final int TOP_MOTOR_FREE_LIMIT_AMPS = 50;
+
+    public static final int BOTTOM_MOTOR_STALL_LIMIT_AMPS = 50;
+    public static final int BOTTOM_MOTOR_FREE_LIMIT_AMPS = 50;
+
+    public static final double RAMP_RATE = .25;
+
+    public static final double TOP_MOTOR_P = 0.0;
+    public static final double TOP_MOTOR_I = 0.0;
+    public static final double TOP_MOTOR_D = 0.0;
+    public static final double TOP_MOTOR_FF = 0.00018;
+
+    public static final double BOTTOM_MOTOR_P = 0.0;
+    public static final double BOTTOM_MOTOR_I = 0.0;
+    public static final double BOTTOM_MOTOR_D = 0.0;
+    public static final double BOTTOM_MOTOR_FF = 0.00018;
+  
+    public static final double RPM_TOLERANCE = 100;
+  }
+
+  public static final class INTAKE {
+    public static final double AXIS_MAX_SPEED = 0.8;
+
+    public static final double DEBOUNCE_TIME_SECONDS = 0.02;
+
+    public static final double FEED_TO_SHOOTER_TIMEOUT = 0;
+    public static final double FLOOR_INTAKE_REVERSE_TIMEOUT = 0.1;
+
+    public static final double PICKUP_SPEED_PERCENT_OUTPUT = .75;
```

Comment:
Remove the constants you are not using in this project

Reply:
done